tests/rptest/services/admin_ops_fuzzer.py

```diff
@@ -458,8 +458,8 @@ def validate_result():
                 try:
                     return op.validate(self.operation_ctx)
                 except Exception as e:
-                    self.redpanda.logger.debug(
-                        f"Error validating operation {op_type} - {e}")
+                    self.redpanda.logger.exception(
```

Comment:
not sure that we want `logger.exception` here - exceptions here are expected and this is going to pollute the info and the main ducktape log.

Reply:
My reasoning is that we were trying to log an exception with `f"... - {e}"` but this method has flaws and often it led to a blank space instead of filling it with useful info. And it isn't redpanda log so we don't fail the test if it has logged exceptions.I don't think pollution is a bad thing in this case: we look at the logs mostly only when they fail and in this case having more information is better than having less.

Reply:
are the semantics that we want here the benefits of `logger.exception(...)` (denis) but avoid polluting the info-level logs since exceptions here are expected (alexey)? if so, then maybe the solution is to do something like `logger.debug(..., exc_info=True)` which I think may satisfy everyone (needs a little quick testing, but this is my understanding of how things work).

Reply:
I like it